framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -74,6 +72,10 @@
     private static final String SOURCES_FILE_ROLE_BINDING = "springboot/kubernetes/role-binding.yaml";
     private static final String SOURCES_FILE_SERVICE_ACCOUNT = "springboot/kubernetes/service-account.yaml";
 
+    private static final String SOURCES_KAFKA_TOPICS_FOLDER = "kafka-topics";
+    private static final String[] SOURCE_KAFKA_TOPIC_FILES =
+            new String[] {"control.yaml", "events.yaml", "kiesessioninfos.yaml", "snapshot.yaml"};
```

Comment:
Can you please add there comment from there are these topics from. It can not be clear in future.